README.md

```diff
@@ -1,70 +1,199 @@
+- Navigate to the directory where you want to clone the project.
 
-### Advanced Configuration
+- Clone the project using by running the following command:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+`git clone https://github.com/yusufyusie/Math-Magicians.git`
+
+### Install
+
+Install this project using the following command: 
+
+`npm install`.
+
+### Usage
+
+To run the project, execute the following commands:
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+  To install all dependencies run the following command:
+
+- npm install
+- `npm run build`
+- `npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+`npm test`
+
+To run Stylelint and ESLint tests, run the following commands respectively:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+```
+npx eslint .
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Yesuf Fenta**
+
+- GitHub: [@yusufyusie](https://github.com/yusufyusie)
+- Twitter: [@yusufyusiee]( https://twitter.com/yusufyusiee)
+- LinkedIn: [yusufyusie](https://www.linkedin.com/in/yusufyusie)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank 
+    - **![](https://img.shields.io/badge/Microverse-blueviolet)**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **How to fix JavaScript + CSS + HTML linter errors?**
+
+  - Kindly use the following in your project's root directory:
+
+  ```
+  npx stylelint "**/*.{css,scss} --fix"
+  ```
+
+  ```
+  npx eslint . --fix
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
```

Comment:
The link to your `mit` license is currently broken, kindly do well to fix it as this currently gives your repo a less professional look😉😉

<img width="759" alt="image" src="https://github.com/yusufyusie/math-magicians/assets/64643995/507053fb-bcdd-4e1a-a6fd-d94a61cb109e">